### DISCUSSION TITLE
Welcome to node-imgur Discussions!

### DISCUSSION THREAD

**Discussion by kaimallea** [kaimallea/node-imgur#120] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by swarajpure** [comment]
Hello there!

I'm Swaraj Rajpure -- an undergraduate in pre-final year who's interested in web development. 

I'm currently developing a web app where you can upload a photo, provide desired size of the photo and you'll get your photo in the required size (no idea about the accuracy of the size yet 😛)

I'm getting the concerned image to my backend server first, resize it and then upload the resized image to imgur and then returning that image link as a response. Why? because I can't have **all** the images stored on the backend server -- due to limitations provided by hosting websites. Imagine my server having 1000 images stored 😂 
After the image is successfully uploaded, I can delete the original and the resized image from the server. Imgur to the rescue! 🚀 

This is going to be a hobby project -- not commercial :)

Good to be here!

**Reply by kaimallea** [reply]
Welcome!
